src/webview/webViewEventHandlers.js

```diff
@@ -150,17 +153,28 @@ const handleImage = (props: Props, src: string, messageId: number) => {
   }
 };
 
-const handleLongPress = (props: Props, _: GetText, isHeader: boolean, messageId: number) => {
-  const message = props.messages.find(x => x.id === messageId);
-  if (!message) {
-    return;
+const handleLongPress = (
+  props: Props,
+  _: GetText,
+  target: string,
+  messageId: number,
+  href: string,
+) => {
+  if (target === 'link') {
+    Clipboard.setString(href);
+    showToast('Link copied to Clipboard');
+  } else {
```

Comment:
This can be made a bit clearer with an early return:
```
if (...) {
  ... showToast(...);
  return;
}

const message = ...
```

Among other things, that keeps the "main story" aligned on a single column, with just the "side notes" indented.

Reply:
I had actually tried that, but `eslint` goes crazy with `eslint(no-useless-return)` and `eslint(no-else-return)`.

Reply:
Hmm -- can you share the exact code that gives you that? (Perhaps in chat, for more-interactive debugging.) We use the early-return pattern often, and don't get those lint errors.

I sometimes have seen lint errors like those on code that was better than the alternative, though; it's possible we should just turn those rules off.

In general, if a linter complains about some code you think is good and doing what the linter says makes it worse, feel free to just disable it locally (preferably like `eslint-disable-line specific/rule` -- `git grep eslint-disable` for examples) and submit the code that way. The disabling will likely be something to discuss, so do mention it in the commit message or the PR.

Reply:
bump - still open

Reply:
Fixed! :)